report-pox-krypton.js

```diff
@@ -310,8 +310,31 @@ const staging_db_path = 'chainstate/chain-00000080-testnet/blocks/staging.db'
 //                                            vtxindex INT NOT NULL
 //     );
 
-const data_root_path = `${root}${process.argv[3] || process.argv[2]}`
-const use_txs = process.argv[2] === '-t'
+// const data_root_path = `${root}${process.argv[3] || process.argv[2]}`
+// const use_txs = process.argv[2] === '-t'
+
+let use_txs = false;
+let use_csv = false;
+var myArgs = process.argv.slice(2);
+// console.log('myArgs: ', myArgs);
+
+for (let j = 0; j < myArgs.length; j++){
+  switch (myArgs[j]) {
+    case '-t':
+      use_txs = true;
+      break;
+    case '-csv':
```

Comment:
convention is to use single letter with `-` and something longer with `--`, so instead, you could have
```
    case '-c':
    case '--csv':
```

Reply:
Fixed in 895d985 and added `--tx-log` for `-t` to be consistent (long and short name for each)